### PR TITLE
Remove experimental feature s

### PR DESCRIPTION
**Pull request by intika**

### CODE REVIEW

**Comment by intika** [comment]
Changes before "v4.0.3 Enhance -H flag" fixed what you asked for in https://github.com/neurobin/shc/pull/74

**Comment by intika** [comment]
**v4.0.3 Enhance -H flag:**
Up to you to merge that change or not... it's an advanced modification that change how radically how shc with H flag works

Basically the resulted binary do: 
- Create /tmp/shc_x.c
- Build it to /tmp/shc_x.so
- LD_PRELOAD the new built library 
- Execute the sh with `sh -c '********' 21<<< "bash content"`
- Remove /tmp/shc_x.so

This is some evil hack lol commands parameters are completely hidden from ps and cmdline 
- The main sh content is at any moment exposed to ps
- The child commands inside the bash are hidden as well (but they are exposed during a small window when they are called before __libc_start_main is modified)
...
**The /tmp/shc_x.c:**
- basically receive the the bash inside 21 with `sh -c '********' 21<<< "bash content"
- inject the bash back as argument in '********' 
- hide bash child arguments by intercepting any child __libc_start_main and changing its argument location

**Comment by intika** [comment]
More explanation about the code here 

https://stackoverflow.com/questions/54819710/hide-execl-arguments-from-ps/

**Comment by intika** [comment]
feel free to reopen this when you have the time or want to merge ;)
